invenio_rdm_records/services/communities/service.py

```diff
@@ -68,14 +73,21 @@ def delete(self, identity, record_id, data, revision_id=None, uow=None):
         for community in communities:
             community_id = community["id"]
             try:
-                self._remove(community_id, record)
+                self._remove(identity, community_id, record)
                 uow.register(RecordCommitOp(record.parent))
                 uow.register(RecordIndexOp(record, indexer=self.indexer))
             except RecordCommunityMissing:
                 errors.append(
                     {
                         "community": community_id,
-                        "message": f"The record {record_id} does not belong to the community {community_id}",
+                        "message": "The record does not belong to the community.",
+                    }
+                )
+            except PermissionDeniedError:
+                errors.append(
+                    {
+                        "community": community_id,
+                        "message": "Permission denied.",
```

Comment:
maybe we could have a more informative message fe. `Your permissions are not sufficient to remove a record from this community`

Reply:
We honestly took what is there everywhere, for consistency. The operation is to remove, to give a meaningful msg we should really explain why the permission is denied e.g. `you are not the owner` or `you are not a curator`